config.go

```diff
@@ -95,7 +95,7 @@ type Config struct {
 
 	ChainPorter tapfreighter.Porter
 
-	BaseUniverse *universe.MintingArchive
```

Comment:
FWIW I think base universe still fits: it's only issuance proofs rn.

Reply:
Also for all other callers: they only care that they're able to register the issuance proofs, the exact naming of the caller doesn't mean much as long as the interface is upheld.

Reply:
I noticed that `MintingArchiveConfig` had a `UniverseForest` field. I was planning on using the `MintingArchive` to update the universe forest (and base universe simultaneously). So I thought it would make sense to generalise the field name `BaseUniverse` to reflect that the forest might be updated also.

Should I not use the minting archive to update the universe forest?

Reply:
>  I was planning on using the MintingArchive to update the universe forest (and base universe simultaneously). So I thought it would make sense to generalize the field name BaseUniverse to reflect that the forest might be updated also.


So the way I imagine it, the minting archive itself doesn't really change much. Instead, the interface that gives it `RegisterIssuance` is what'll be swapped out to instead point to the on-disk multi-verse, that handles all the normal issuance stuff, but then also updates the multiverse tree.

Reply:
I've reverted this change.